src/containers/Tablet/components/TabletStorageInfo/TabletStorageInfo.scss

```diff
@@ -0,0 +1,71 @@
+@import '../../../../styles/mixins.scss';
+
+.ydb-tablet-storage-info {
```

Comment:
I've thought about this, but decided to wait for third case. After that it will be more clear, what styles are common!
So it's 100% a must, I'll definitely do it a bit later.

Reply:
Okay, let's not forget about it further